### DISCUSSION TITLE
how to config topk?

### DISCUSSION THREAD

**Discussion by bingbb47** [milvus-io/milvus#21844] [Q&A and General discussion] [answered]
https://github.com/milvus-io/milvus/pull/21155  Made TopK limit configurable
but how to specify the value of topk
I hope to return top 30000

**Comment by bingbb47** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/111261344/215269320-b4274c7f-f3d6-4884-82e6-47c9db3fdf70.png)

**Comment by yhmo** [comment]
The #21155 read the topk limit by this line:` p.Base.ParseInt64WithDefault("common.topKLimit", 16384)`
"common.topKLimit" means read the value from **milvus.yaml**, by the "topKLimit" under the "common" section.
So you can manually add "topKLimit" under the "common":
```
common:
  topKLimit: 30000
  ......
```
Restart the service after you do the change.
